### PR TITLE
[🔥AUDIT🔥] Add a job to update the dev-server static images in gcloud.

### PR DESCRIPTION
**Pull request by csilvers**
🖍 _This is an audit!_ 🖍

## Summary:
When called (which we expect to be every day), this job rebuilds
all the images that we use for local dev-server -- one for each
first-party service in webapp -- and uploads them to the google
container registry.  It then updates a file in webapp saying what
git commit corresponds to the latest-pushed images, and checks
that into the `automated-commits` branch.

Issue: https://khanacademy.atlassian.net/browse/INFRA-9151

## Test plan:
`groovy jobs/update-devserver-static-images.groovy` gave only
the expected two errors around `@Library`.

Will try running it and see what happens!

### CODE REVIEW

File: jobs/update-devserver-static-images.groovy [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+   kaGit.safeMergeFromMaster("webapp", params.GIT_BRANCH);
+
+   dir("webapp") {
+      sh("timeout 2m git clean -ffd"); // in case the merge modified .gitignore
+      sh("timeout 1m git clean -ffd"); // sometimes we need two runs to clean
+
+      sh("make fix_deps");  // force a remake of all deps all the time
+
+      // Run the script!
+      sh("Make -C dev/server upload-all-static-images");
```
Comment (dbraley) [inline jobs/update-devserver-static-images.groovy:51]: Does capital 'M' `Make` work?

**Review by dbraley — APPROVED** [review]
I think this is the same as the other job you had, other than the one file name change?

Though I didn't think the `make` command was normally case-insensitive, so maybe a small fix there? But LGTM otherwise (and that's fine if it works too).

File: jobs/update-devserver-static-images.groovy (hunk 1 above)
Comment (csilvers) [reply jobs/update-devserver-static-images.groovy:51]: Nope!  I had to fix that later.

**Comment by csilvers** [comment]
It's unclear to me why this is still open when it's an audit!  And it's definitely been merged.  Weird.

**Comment by csilvers** [comment]
Oh, looks like it was landed in #126.  I have no idea how this dup got created.

**Pull request closed without merging**